src/com/l4gunner4l/reader/FileEmployeeReader.java

```
@@ -0,0 +1,54 @@
+package com.l4gunner4l.reader;
+
+import com.l4gunner4l.base.BaseEmployeeReader;
+import com.l4gunner4l.model.AverageEmployee;
+import com.l4gunner4l.model.Employee;
+import com.l4gunner4l.model.HourEmployee;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+
+public class FileEmployeeReader extends BaseEmployeeReader {
+
+    public static final String SEPARATOR = " ";
+
+    public FileEmployeeReader() {
+    }
+
+    public ArrayList<Employee> readEmployees(String path) {
+        try (BufferedReader reader = new BufferedReader(new FileReader(new File(path)))) {
+            ArrayList<Employee> emps = new ArrayList();
+            String line;
+            while ((line = reader.readLine()) != null) {
+                try {
```

Предлагаю это исключение убрать. Fail fast